### PR TITLE
Pigeoncoop

### PR DESCRIPTION
**Pull request by corydodt**

### CODE REVIEW

File: smokesignal.py [hunk 1]
```diff
@@ -15,6 +15,21 @@
...
+_call_partial = None
+
+def install_twisted():
+    """
+    If twisted is available, make `emit' return a DeferredList
+    """
+    global emit, _call_partial
+    try:
+        from twisted.internet import defer
+        emit = _emit_twisted
```
Comment (alangrafu) [inline smokesignal.py]: Twisted 14.0 is recommended but I wonder is there is a minimum version number aceptable? Should we include that?

Comment (corydodt) [inline smokesignal.py]: IDK what the minimum would be. DeferredList and maybeDeferred are the only APIs I'm aware of, and those have existed for eons. But I've only tested with 14.0, so I'm not going to make any claims about older versions, lest we are compelled to fix bugs in those combinations.

Comment (alangrafu) [inline smokesignal.py]: The I think we should leave at least a comment indicating that this works with 14.0 and above, just to make it clear

Comment (corydodt) [inline smokesignal.py]: OK, updated - I added it to the docstring.

**Pull request merged**